Following bug report instructions in `bug-report-test.ts` leads to: `Command "bug-report-test" not found`

### I'm using React Router as a...

framework

### Reproduction

Go to https://github.com/jonbri/react-router and run the commands specified in `bug-report-test.ts`:
```sh
pnpm install && pnpm build
pnpm bug-report-test
```

### System Info

```shell
`https://github.com/jonbri/react-router` is a fork with no changes.

This is using WSL2 Ubuntu.


jonbri@l10g878:~/a/react-router$ npx envinfo --system --npmPackages '{vite,react-router,@react-router/*}' --binaries --browsers
Need to install the following packages:
envinfo@7.14.0
Ok to proceed? (y) y


  System:
    OS: Linux 5.15 Ubuntu 22.04.4 LTS 22.04.4 LTS (Jammy Jellyfish)
    CPU: (8) x64 Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz
    Memory: 5.41 GB / 7.62 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 20.18.1 - ~/.n/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.8.2 - ~/.n/bin/npm
    pnpm: 9.15.4 - ~/.n/bin/pnpm
    bun: 1.0.10 - ~/.bun/bin/bun
  Browsers:
    Chromium: 132.0.6834.83
  npmPackages:
    vite: ^6.0.0 => 6.0.2
```

### Used Package Manager

pnpm

### Expected Behavior

Running `pnpm bug-report-test` should complete since no changes have been made.

### Actual Behavior

An error occurs:
```
jonbri@l10g878:~/a/react-router$ pnpm bug-report-test
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "bug-report-test" not found
```

--- a/integration/bug-report-test.ts
+++ b/integration/bug-report-test.ts
@@ -12,8 +12,7 @@ let fixture: Fixture;
 let appFixture: AppFixture;
 
 ////////////////////////////////////////////////////////////////////////////////
-// 💿 👋 Hola! It's me, Dora the Remix Disc, I'm here to help you write a great
-// bug report pull request.
+// 👋 Hola! I'm here to help you write a great bug report pull request.
 //
 // You don't need to fix the bug, this is just to report one.
 //
@@ -24,23 +23,30 @@ let appFixture: AppFixture;
 // commit to this pull request, and your now-succeeding test will have to be moved
 // to the appropriate file.
 //
-// First, make sure to install dependencies and build Remix. From the root of
+// First, make sure to install dependencies and build React Router. From the root of
 // the project, run this:
 //
 //    ```
 //    pnpm install && pnpm build
 //    ```
+// 
+// If you have never installed playwright on your system before, you may also need
+// to install a browser engine:
+//
+//    ```
+//    pnpm exec playwright install chromium
+//    ```
 //
 // Now try running this test:
 //
 //    ```
-//    pnpm bug-report-test
+//    pnpm test:integration bug-report --project chromium
 //    ```
 //
 // You can add `--watch` to the end to have it re-run on file changes:
 //
 //    ```
-//    pnpm bug-report-test --watch
+//    pnpm test:integration bug-report --project chromium --watch
 //    ```
 ////////////////////////////////////////////////////////////////////////////////
 
@@ -94,7 +100,7 @@ test.afterAll(() => {
 
 ////////////////////////////////////////////////////////////////////////////////
 // 💿 Almost done, now write your failing test case(s) down here Make sure to
-// add a good description for what you expect Remix to do 👇🏽
+// add a good description for what you expect React Router to do 👇🏽
 ////////////////////////////////////////////////////////////////////////////////
 
 test("[description of what you expect it to do]", async ({ page }) => {
@@ -116,5 +122,6 @@ test("[description of what you expect it to do]", async ({ page }) => {
 });
 
 ////////////////////////////////////////////////////////////////////////////////
-// 💿 Finally, push your changes to your fork of Remix and open a pull request!
+// 💿 Finally, push your changes to your fork of React Router 
+// and open a pull request!
 ////////////////////////////////////////////////////////////////////////////////

